### ISSUE TITLE
calls to functions within a contract

### ISSUE THREAD

**Issue by vd1** [jkrivine/tl_interpreter#3]
It seems that at the moment in a call to a function defined within a contract module, one cannot pass more than one argument:
```
tx uAlice cA C0.provision (1,true) 
```
works but not
```
tx uAlice cA C0.provision 1 true
```

**Comment by adhusson** [comment]
Yes, multiple arguments should be given as a tuple. I don't know how to let tx accept a varying number of arguments.
